intl/msg_hash_us.h

```diff
@@ -209,11 +209,11 @@ MSG_HASH(
       )
 MSG_HASH(
       MENU_ENUM_LABEL_VALUE_AUTO_OVERRIDES_ENABLE,
-      "Load Override Files Automatically"
+      "Load Core Overrides Automatically"
```

Comment:
Is this game-specific override?
```
 -      "Load Content-specific core options automatically")
 +      "Load Content Overrides Automatically")
```
Based on recent changes from `Load Configuration, Save Current Config and Save New Config`to all *`Configurations`, I assume to go with that to stay consistent. It's also 4 chars longer than `Directory` trimmed to `Dir`.
```
Configuration... to... Config    -7 chars
Directory... to... Dir    -6 chars
```
How do you override the config? With core config or is it something else?

Pinging @fr500 for feedback on this one. He's tough on me.